src/vm/amd64/unixasmhelpers.S

```diff
@@ -27,7 +27,7 @@ LEAF_ENTRY PrecodeFixupThunk, _TEXT
         lea     METHODDESC_REGISTER,[rax+r11*8]
 
         // Tail call to prestub
-        jmp     ThePreStub
+        call C_FUNC(ThePreStub)
```

Comment:
This does not look right. The call instruction will make the stack aligned incorrectly, right? Could this be the root cause of the issue in ThePreStub that you and @janvorli were discussing earlier?